lib/Echidna.hs

```diff
@@ -19,12 +19,11 @@ import EVM.Solidity (BuildOutput(..), Contracts(Contracts))
 import EVM.Types hiding (Env)
 
 import Echidna.ABI
-import Echidna.Etheno (loadEtheno, extractFromEtheno)
```

Comment:
is `Etheno.hs` still needed? if not, can you remove it?

Reply:
Done: removed code and tests